### PR TITLE
Math Magician: Components

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this Pull Request, I: 

- [x]  Delete all the boilerplate from CRA(text, images, styles).
- [x] Create a component directory.
- [x] Create a new Calculator component in the components directory matching this design 

image

- [x] Style the Calculator component with Calculator.css. 
- [x] Make the Calculator a class-based component.
- [x] Import and Render the Calculator component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <Calculator />
```
Comment (Davidosky007) [inline src/App.js:6]: * Kindly Use class-based components here as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.).

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Ntwali-Josue 👋, 

Good job so far! 💪 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
* Proper use of git-flow ✔️ 
## There are some issues that you still need to work on to go to the next project but you are almost there!
almost-gif


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Ntwali-Josue) [reply src/App.js]: Oh, @Davidosky007 thanks for showing me this 😃 , I am going to work on it.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉 

# Hi @Ntwali-Josue, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ntwali-Josue** [comment]
@DeliceLydia thanks a lot for approving my project.

**Pull request merged**